### ISSUE TITLE
Does not work with Kiwi Browser

### ISSUE THREAD

**Issue by GG700** [sbs20/scanservjs#180]
Hi,

**- With a smartphone (Android, Chrome or chromium)** 
It's strange, but the item "device" don't indicates that I have selected a scanner (I have tested with one scanner). It indicates "Unspecified". But if I clic on "SCAN", the scanning operation proceeds normally.
It was ok in version 2.1. 

**- With a computer (Linux, firefox)** 
It's ok.
I am able to find my device.

**Comment by sbs20** [comment]
Hi. Which version are you using? Can you send a screenshot?

**Comment by GG700** [comment]
In fact, it is impossible to scan with the smartphones if I stop I restart the scanner (in this case, the number of libusb changes; and so the smartphone tried already to scan with the old "adress").
I confirm It is ok with my personal computer.

**scanservjs** : V2.8.4

**My first smartphone : 
SAMSUNG GLX A3 2016**
Android : 7.0
Browser : GGLE CHROME last update (22/3/21)

**The second : 
SAMSUNG A40**
Android : 9
Browser : KIWI (chromium) 22/03/21

![GLX_A3_2016_20210411-101704](https://user-images.githubusercontent.com/49592541/114297261-eb25a800-9aaf-11eb-9548-24c1211376b2.png)

**Comment by sbs20** [comment]
If the USB device ID changes then that is meant to be fixed by pressing reset. Once you reset it should find and list your device. Does that not happen?

**Comment by sbs20** [comment]
Does it work if you go incognito?

**Comment by GG700** [comment]
With my PC, it is ok when pressing RESET.

I progress.
It is ok with my smartphone with GOOGLE CHROME.
It is not ok with my smartphone with the browser KIWI BROWSER (chromium) : 
I can scan but I can't modify every item (resolution, mode, Batch, etc...). the GUI does not recognize the scanner. But in the background, scanservjs manages to recognize the change of USB ID ... !!! 

Nota : all the extensions of Kiwi browsers are disabled in the test (adblock,...).

**Comment by sbs20** [comment]
The UI uses local storage to remember data. I wonder if kiwi is disabling that by default? I've tested with chrome and Firefox on Android and it seems to work.

**Comment by GG700** [comment]
I don't know how to verify if Kiwi UI uses local storage to remerber datas.
But I know that before (V2.1), it was ok. I there a difference for this proint between v2.1 and now ?

**Comment by sbs20** [comment]
I have managed to reproduce this. And I managed to do some rudimentary debugging but there's nothing obvious going on. The requests and responses between the browser and server are intact - certainly, nothing is being blocked -, but something about the page rendering is not working. The big change between 2.1 and now is using a different UI framework (vuetify over bootstrap) - I can only assume it's something there since nothing else much has changed.

I'm afraid, since it's working in Chrome, Firefox and Safari, I don't think I can dedicate any further time to making this work. Sorry. If someone else manages to find a fix then I am very open to merging it.

**Comment by sbs20** [comment]
I actually stumbled on the answer to this as part of #243 

Kiwi is most likely built on a version of Chromium which is in line with Chrome 84. That in turn was unable to use certain newer language features. I had thought that polyfills for missing language features were generated automatically but I was incorrect. I just confirmed by installing Kiwi and running against a new dev build.

This will be resolved with the linked issue.
